packages/k8s/k3s-openrc/collection.yaml

```diff
@@ -1,7 +1,7 @@
 packages:
   - name: k3s
     category: k8s
-    version: "1.25.11+1"
+    version: "1.25.11"
```

Comment:
This wont work no? the package with +1 will be the latest in luet eyes I think @mudler can confirm?

Reply:
We split the package into 2 new ones. So for luet, this is the first time it sees this (new) package.

But...we forgot to change the name of the package. Doing that now.